tinytuya/core.py

```diff
@@ -1391,13 +1391,27 @@ def set_value(self, index, value, nowait=False):
 
         return data
 
+    def set_multiple_values(self, data, nowait=False):

```

Comment:
Love this.  Was on my old TODO so thank you! ;) I also think there was an issue for this - I'll see if I can find it.